### PR TITLE
Multiple files 

### PR DESCRIPTION
**Pull request by ajb**
cf https://github.com/dobtco/screendoor-v2/issues/477

### CODE REVIEW

**Comment by ajb** [comment]
See https://github.com/dobtco/screendoor-v2/pull/3061 for the main discussion. The main thing to note in this PR is that we now include basic button styles in this repo, because:

1. We already discussed adding some beefier-styles, and
2. I needed a way to add styling to the `<label>` element that triggers the file upload

File: src/templates/fields/file.eco [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+
+<% if @model.canAddFile(): %>
+  <div class='fr_add_file'>
+    <label for='<%= @getDomId() %>' class='<%= FormRenderer.BUTTON_CLASS %>'>
+      <%= if @model.getFiles().length then FormRenderer.t.upload_another else FormRenderer.t.upload %>
+    </label>
+
+    <input type='file'
+           id='<%= @getDomId() %>'
+           style='position:fixed;left:-9999px;'
```
Comment (jrubenoff) [inline src/templates/fields/file.eco]: This is a hack. Can we apply [these styles](https://github.com/dobtco/dvl-core/blob/master/vendor/assets/stylesheets/dvl/core/includes.scss#L326) instead?

**Comment by jrubenoff** [comment]
Button text is turning gray on hover, when it should stay white:

button

Other than that, looks great.

**Comment by ajb** [comment]
Thanks! Moved the inline styles to the stylesheets and fixed the bug in the preview page that was leading to those issues w/ button hover.

**Pull request merged**